src/validators/redteam.ts

```diff
@@ -90,6 +90,7 @@ export const RedteamGenerateOptionsSchema = z.object({
   purpose: z.string().optional().describe('Purpose of the redteam generation'),
   strategies: z.array(RedteamStrategySchema).optional().describe('Strategies to use'),
   write: z.boolean().describe('Whether to write the output'),
+  delay: z.number().int().positive().optional().describe('Delay in milliseconds generators'),
```

Comment:
```suggestion
  delay: z.number().int().positive().optional().describe('Delay in milliseconds for generators'),
```

Reply:
why are there 3 places where this string has to be copied?  are the zod descriptions ever shown to the user?